.github/workflows/ci.yml

```diff
@@ -67,6 +67,7 @@ jobs:
         -   uses: actions/checkout@v4
         -   name: "Main Script"
             run: |
+                export PROJECT_INSTALL_FLAGS="-v"
```

Comment:
@inducer: can we make `-v` the default in ci-support? This greatly helps when trying to debug failures that happen because the underlying build system changes in some way (like in this case).

Reply:
Sure. Could you do a PR?

Reply:
https://github.com/inducer/ci-support/pull/39